3_javascript/javascript1/week2/homework javascript1 week2/weather.js

```diff
@@ -0,0 +1,16 @@
+//Weather wear //
+
+function weatherWear(temperature) {
+  let clothesToWear = "jackets, sweaters";
+  clothesToWear = "jackets, sweaters";
+  if (temperature <= 10) {
```

Comment:
Good use of `else if`! 🚀 

The function could be a bit simplified with moving the value of  `clothesToWear` directly into the string being returned.

Reply:
Did that already.